src/libs/Navigation/Navigation.ts

```diff
@@ -520,49 +520,51 @@ function navigateToReportWithPolicyCheck(
     forceReplace = false,
     ref = navigationRef,
 ) {
-    const targetReport = reportID ? {reportID, ...allReports?.[`${ONYXKEYS.COLLECTION.REPORT}${reportID}`]} : report;
-    const policyID = policyIDToCheck ?? getPolicyIDFromState(navigationRef.getRootState() as State<RootNavigatorParamList>);
-    const policyMemberAccountIDs = getPolicyEmployeeAccountIDs(policyID);
-    const shouldOpenAllWorkspace = isEmptyObject(targetReport) ? true : !doesReportBelongToWorkspace(targetReport, policyMemberAccountIDs, policyID);
+    isNavigationReady().then(() => {
```

Comment:
This change was approved by @WojtekBoman himself 🙇

Reply:
the man, the legend